stream/src/main/scala/org/apache/pekko/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -62,6 +62,8 @@ import pekko.util.OptionVal
 
   val Debug = false
 
+  val MailboxConfigName: String = "pekko.stream.materializer.mailbox"
```

Comment:
This is hard coding, I don't know if there is a place for  processing configaration specially?
like  `org.apache.pekko.stream.ActorAttributes#IODispatcher` ?

Reply:
It's read by typesafe config